test/controllers/PackAtBusinessAddressControllerSpec.scala

```diff
@@ -108,6 +118,8 @@ class PackAtBusinessAddressControllerSpec extends SpecBase with MockitoSugar {
         val result = route(application, request).value
 
         status(result) mustEqual SEE_OTHER
+        redirectLocation(result).value mustEqual onwardRoute.url
```

Comment:
What part of the code is this covering? If I where to break something that this test line would catch what would it be?

Reply:
left in by lack of attention.  Removed as it has no function.